### DISCUSSION TITLE
Fusion DTU löscht "Gesamtertrag Heute" abends nach automatischem WR-Abschalten wegen Dunkelheit

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2136] [Q&A]
Hallo,

ich habe mein PV-System kürzlich erweitert und betreibe nun seit über 2 Wochen eine Fusion OpenDTU mit dem HMS-1000 (als WR-1) und dem HM-800 (als WR-2).

Alles läuft nach anfänglichen, kleineren Problemchen prima. Aus reinem Interesse für die neue Anordnung habe ich mir in den letzten Tagen öfter mal die DTU-Daten angesehen, wobei dabei dann folgends Problem aufgefallen ist.

Zunächts noch normales Verhalten.
Bei abnehmender Helligkeit (abends) wechselt ein WR von blau nach orange, etwas später dann der andere.
Nach einiger Zeit wechselt dann ein WR nach rot (welcher zuerst, hängt von der restlichen Einstrahlung ab) und nach einiger Zeit dann auch der andere, so dass beide schließlich den Status "rot" haben. 

Aber dann passiert Folgendes:
Der HMS-1000 (WR-1) wechselt kurz zurück von rot nach orange und dann wieder zu rot. Bei diesen Wechselvorgängen wird der Wert bei "Gesamtertrag Heute" im HMS-1000 auf Null gesetzt und bleibt da auch so.
Der HM-800 verhält sich weiterhin immer ok..

Ich habe in den letzten 5 Tagen das Verhalten beobachtet und es ist jeden Abend identisch, das kann also kein Zufall sein.

Alle SW ist aktuell. Die Aktualisierung hatte ich mal auf 1 sec gestellt, aber dann wieder zurück auf 3 sec, ohne Auswirkungen.

Ich kenne mich in der SW nicht aus und bin nun völlig ratlos, was ich noch testen, ändern oder untersuchen soll.

Ich bitte daher hier um hilfreiche Unterstützung der Wissenden - vielen Dank im Voraus!

Gruß
Werner

**Comment by ms49434** [comment]
Schau mal bitte nach, ob in den Einstellungen der Wechselrichter die Option Nulle Laufzeitdaten aktiv ist.
![image](https://github.com/tbnobody/OpenDTU/assets/94918347/9be72fa4-38cf-4157-8835-cc6d1d00063e)

**Comment by Fokker-1** [comment]
Hallo ms49434,

danke für die Info.

Die Einstellungen beim Problemfall HMS-1000 (WR-1) sind:
- "Nulle Laufzeit Daten": inaktiv
-  "Nulle Tagesertrag um Mitternacht": inaktiv
- "Lösche Ereignisanzeige um Mitternacht": aktiv
-  "Erreichbarkeit Schwellenwert": 5 s

Beim HM-800 (WR-2):
- "Nulle Laufzeit Daten": inaktiv
-  "Tagesertrag um Mitternacht": inaktiv
- "Lösche Ereignisanzeige um Mitternacht": inaktiv
-  "Erreichbarkeit Schwellenwert": 2 s

Ich hoffe, das kann helfen.

Danke und Gruß
Werner

**Reply by Fokker-1** [reply]
Hallo,

ich hatte gestern beim HMS-1000 (WR-1) mal diese Einstellungen geändert:

- "Lösche Ereignisanzeige um Mitternacht": von aktiv in inaktiv
- "Erreichbarkeit Schwellenwert": von 5 in 2.

Damit sind diese Einstellungen in beiden WR nun gleich.

Gestern Abend ist das Problem mit dem Reset von "Gesamtertrag Heute" nicht (!) aufgetreten.
Das kann natürlich auch ein Zufall sein, ich werde das weiter beobachten in den nächsten Tagen,

Allerdingd habe ich auch keine Idee, warum die o. g. Änderungen das Problem beheben sollten, ich sehe da keinen Zusammenhang.

Ich melde mich dazu wieder.

Gruß
Werner

**Reply by Fokker-1** [reply]
Kurzes Update:

Das Problemi ist nun 3 Abende in Folge nicht mehr aufgetreten.
Trotzdem: ich finde keinen Zusammenmhang zwischen den Änderungen (siehe oben):

  - "Lösche Ereignisanzeige um Mitternacht": von aktiv in inaktiv
  - "Erreichbarkeit Schwellenwert": von 5 in 2.

Gruß
Werner

**Comment by tbnobody** [comment]
Bitte die Doku lesen: https://www.opendtu.solar/firmware/faq/#basics

![image](https://github.com/tbnobody/OpenDTU/assets/1753222/f0f6e6e2-e45a-4de2-ba6a-fe86c9224692)

**Reply by Fokker-1** [reply]
Danke für die Info.

Sicher, das kann so natürlich immer grundsätzlich möglich sein.
Die Frage ist dann aber auch, mit welcher Wahrscheinlichkeit passiert das an 5 Tagen in Folge?

Um diese Zeit kurz vor 22 Uhr gab es auch keine stärkeren Wolkenbewegungen, bei denen noch einmal merkliche Helligkeit auf die Panels hätte fallen können. Der Übergang rot => orange => rot lag geschätzt im Bereich < 30 sek.

Wie auch immer, ich werde das noch weiter beobachten (wenn es zeitlich klappt) und dann hier wieder berichten.

Andere Frage noch: 
Kann man die Reihenfolge der beiden Inverter verändern, ohne die Daten alle neu eingeben zu müssen?

Danke und Gruß
Werner

**Comment by ms49434** [comment]
Na klar kann man die Reihenfolge verändern, einfach den Inverter über die Pfeile verschieben, dann auf Reihenfolge abspeichern klicken:
![image](https://github.com/tbnobody/OpenDTU/assets/94918347/deba220a-5f27-4744-84a2-6220412c7ec7)

**Reply by Fokker-1** [reply]
Hallo,

danke für die Info.

Ja, genau so habe ich das auch erwartet, aber es ist mir in der Vergangenheit nicht gelungen.
Tippen, Halten, Schieben auf dem Handy haben da nichts bewegt, nur der Handy-Screen wurde hin und her geschoben.

Da es ja ganz offensichtlich trotzdem gehen muss, habe ich es jetzt mal am PC versucht.
Eine Veränderung über die Pfeile hat nicht geklappt.

Aber hält man mit gedrückter Maustaste das horizontale Symbol in der Spalte "#" fest, kann man darüber die Zeile auf diese Weise verschieben.

Vielleicht wäre dort so ein "i" - Button mit dieser Info an dieser Stelle hilfreich - danke!

Wieder was gelernt!

Gruß
Werner

**Comment by Fokker-1** [comment]
Hallo,

ich komme jetzt noch mal auf das Problem zurück, dass bei meinem HMS-1000 in der Dämmerung die Daten "Tagesproduktion" gelöscht werden.

Das habe ich mir nun einige Abende mal genauer angesehen mit folgenden Ergebnissen:

1. Übergang von Blau auf Orange mit zunehmender Dämmerung
2.  im Status Orange gibt es z. T. mehrere Wechsel wieder zurück auf Blau, diese dauern aber i.d.R immer nur wenige Sekunden
3. danach folgt ein Zustand, bei dem Orange einige Zeit stabil stehen bleibt
4. dann erfolgt der Wechsel nach ROT, soweit ist das ja auch ok.
5. nun kommt es vor, dass vom Status Rot wieder zrück gesprungen wird in den Status Orange.
    Dies kann auch bis zu 5x erfolgen, wobei die Dauer des Zustandes Orange dann immer nur im Sekundenbereich liegt (ca. <15 sec)
6. Hier hat es in der Vergangenheit dann das Problem gegeben, dass dabei die die "Tagesproduktion" auf "0" gesetzt wurde (geschätzt in ca. 3/4 aller solcher Fälle), was ja grundsätzlich auch dem erwarteten (aber hier unerwünschten) Verhalten bei einem solchen Übergang entspricht
7. ich hatte nun bei "Wechselrichter bearbeiten" die "Tagesertragskorrektur" aktiviert, nach dem ich dazu einen interessanten Beitrag bei GitHub gelesen hatte. Vorher konnte ich damit nicht wirklich was anfangen.
8. gestern nun habe ich das genauer beobachtet: es gab 5 Übergänge von Rot zurück auf Orange, immer nur einige Sekunden. Eine Löschung der "Tagesertragskorrektur" trat nicht auf. Ich hoffe, das Problem konnte damit behoben werden. ich melde mich dazu in einigen Tagen mit den Ergebnissen noch mal.
9. es ist schon etwas erstaunlich, die Panels zeigen zu keiner Straße, kleinere Häuser stehen auch erst in ca. 70 m Entfernung. Vielleicht hat der HMS-1000 bei seinen Umschaltwerten ja keine Hystere eingebaut, so dass geringste Erhöhungen / Verringerungen der Einstrahlung diesen Effekt des Hin- und Herschaltens auslösen.

Gruß
Werner

**Comment by Fokker-1** [comment]
Hallo,

zum Themenkomplex noch eine weitere Frage:

Welche Auswirkungen auf das Verhalten der DTU hinsichtlich Umschaltung, Messwerterzeugung usw. haben die Dämmerungseinstellungen?
Was die Bezeichnungen allgemein bedeuten, weiß ich wohl, aber nicht hinsichtlich des DTU-Verhaltens.

Danke und Grüße
Werner

**Reply by ms49434** [reply]
Diese Einstellungen wirken sich auf die Kommunikation mit dem Inverter aus, siehe https://www.opendtu.solar/firmware/configuration/inverter_settings/#poll-inverter-data-at-night und https://www.opendtu.solar/firmware/configuration/inverter_settings/#send-commands-at-night. Einige Optionen in der Web-GUI sind im Nachtmodus nicht verfügbar.

**Reply by Fokker-1** [reply]
Hallo,

danke für den Link mit den hilfreichen Infos.

Hinsichtlich der "Dämmerung" steht dort:

"Sunset type

There are differnt "Sunset types" which add different offsets to the calculated time. The following values are possible:

    Standard dawn (90.8°)
    Nautical dawn (102°)
    Civil dawn (96°)
    Astronomical dawn (102°)"

Die verschiedenen Zustände sind mir klar.
Nicht klar ist mir (leider), welche konkreten Auswirkungen "different offsets to the calculated time" solche Änderungen zur Folge haben.

Für eine kurze Info wäre ich dankbar.

Viele Grüße
Werner
